src/sicks3000.cc

```diff
@@ -81,7 +81,8 @@ int SickS3000::Open()
   // Setup serial device
   if (this->serial->OpenPort2() == SERIAL_ERROR)
   {
-    ROS_ERROR("SickS3000::Open: Error Opening Serial Port");
+    ROS_ERROR_ONCE("SickS3000::Open: Error Opening Serial Port");
```

Comment:
The theory is that an error which prevents the port from opening is probably permanent.